docs/conf.py

```diff
@@ -90,12 +90,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = [
-    "_build",
-    "samples/AUTHORING_GUIDE.md",
-    "samples/CONTRIBUTING.md",
-    "samples/snippets/README.rst",
-]
```

Comment:
This change looks odd to me:  is `samples/*` already excluded in some other way?

Reply:
In fact, this part of the change causes the `docs` build to fail:

```
sphinx.errors.SphinxWarning: /tmpfs/src/github/python-bigquery/docs/samples/AUTHORING_GUIDE.md:document isn't included in any toctree
```

@busunkim96, @software-dov It this due to a change in the template used in the macrogenerator?  If so, I guess we need to drop copying `docs/conf.py` during a synth run.

Reply:
@tseaver I think excluding is the correct thing to do since BigQuery needs a different set of configs